idb/dummy.go

```diff
@@ -335,6 +335,9 @@ type AccountQueryOptions struct {
 	IncludeAssetHoldings bool
 	IncludeAssetParams   bool
 
+	// IncludeDeleted indicated whether to include deleted Assets, Applications, etc within the account.
```

Comment:
leaving a comment here as a reminer in case we decide to have the original names.